CMakeLists.txt

```diff
@@ -370,7 +370,9 @@ OPTION(ENABLE_TCMALLOC "Enable TCMalloc support" OFF)
 if (ENABLE_TCMALLOC)
   message(STATUS "TCMalloc support enabled")
   set(TCMALLOC_HEADER "gperftools/malloc_extension.h")
-  check_include_file_cxx("${TCMALLOC_HEADER}" HAVE_GPERFTOOLS_MALLOC_EXTENSION_H)
+  find_path(TCMALLOC_INCLUDE_DIR "${TCMALLOC_HEADER}")
+  set (CMAKE_REQUIRED_INCLUDES "${TCMALLOC_INCLUDE_DIR}")
+  check_include_file_cxx("${TCMALLOC_HEADER}" HAVE_GPERFTOOLS_MALLOC_EXTENSION_H )
```

Comment:
There's an additional space before the parenthesis `)`.

Reply:
Yes. Just for consistency with the rest of the code this should be `set(CMAKE_REQUIRED_INCLUDES ...)`.